core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -0,0 +1,181 @@
+package org.ensime.core
+
+import scala.concurrent.duration._
+
+import akka.actor._
+import akka.event.LoggingReceive.withLabel
+import org.ensime.api._
+import org.ensime.config.richconfig._
+import org.ensime.util.{ Debouncer, Timing }
+import org.ensime.util.FileUtils.toSourceFileInfo
+import org.ensime.util.file._
+
+final case object SuspendAnalyzer
```

Comment:
this should be inside `Analyzer`, not the manager. The manager doesn't need to know anything about the analyzers suspending themselves.